trunk/source/CACSL2BoogieTranslator/src/de/uni_freiburg/informatik/ultimate/plugins/generator/cacsl2boogietranslator/Boogie2ACSL.java

```diff
@@ -134,7 +134,7 @@ public BacktranslatedExpression translateEnsuresExpression(
 
 	private Optional<Expression> computeOldVarEqualities(final String proc,
 			final Set<de.uni_freiburg.informatik.ultimate.boogie.ast.IdentifierExpression> modifiableGlobals) {
-		final var modifiableNames = modifiableGlobals.stream().map(e -> e.getIdentifier()).collect(Collectors.toSet());
+		final var modifiableNames = modifiableGlobals.stream().map(de.uni_freiburg.informatik.ultimate.boogie.ast.IdentifierExpression::getIdentifier).collect(Collectors.toSet());
```

Comment:
`e -> e.getIdentifier())` is definitely nicer than `de.uni_freiburg.informatik.ultimate.boogie.ast.IdentifierExpression::getIdentifier`...
Also, Eclipse does not respect the maximum line length here.

Reply:
Maybe even without this change, we should run `Clean up` twice.

Reply:
Up to the fixpoint :)